src/test/mediapipeflow_test.cpp

```diff
@@ -2839,6 +2847,453 @@ TEST_F(MediapipeFlowStartTest, AsSoonAsMediaPipeGraphDefinitionReadyInferShouldP
     t.join();
 }
 
+std::unordered_map<std::type_index, ovms::Precision> TYPE_TO_OVMS_PRECISION{
+    {typeid(float), ovms::Precision::FP32},
+    {typeid(uint64_t), ovms::Precision::U64},
+    {typeid(uint32_t), ovms::Precision::U32},
+    {typeid(uint16_t), ovms::Precision::U16},
+    {typeid(uint8_t), ovms::Precision::U8},
+    {typeid(int64_t), ovms::Precision::I64},
+    {typeid(int32_t), ovms::Precision::I32},
+    {typeid(int16_t), ovms::Precision::I16},
+    {typeid(int8_t), ovms::Precision::I8},
+    {typeid(bool), ovms::Precision::BOOL},
+    {typeid(double), ovms::Precision::FP64},
+    {typeid(void), ovms::Precision::BIN}};
+
+template <typename T>
+std::vector<T> prepareData(size_t elemCount, T value, bool maxValue) {
```

Comment:
Is it possible to set max as default for `value` ? It could simplify the function.